torquedata/torquedata/routes.py

```diff
@@ -52,21 +52,21 @@ def search(sheet_name):
 
 @app.route('/api/sheets')
 def sheets():
-    """ Return a summary of all the sheets """
+    """Return a summary of all the sheets"""
     group = request.args.get("group")
     wiki_key = request.args.get("wiki_key")
 
     response = []
     for sheet_name, sheet in data.items():
-        sheet = sheet.apply_permissions(permissions[sheet_name][wiki_key][group])
```

Comment:
We now no longer call `sheet.apply_permissions()`, so I would expect to see the permissions check happening manually somewhere in the new code introduced in this commit... but I don't see that anywhere?